pages/linux/slapt-get.md

```diff
@@ -0,0 +1,28 @@
+# slapt-get
+
+> An apt like system for Slackware package management.
+> Package sources need to be configured on the slapt-getrc file.
+
+- Update the list of available packages and versions (it's recommended to run this before other `slapt-get` commands):
```

Comment:
Not sure that the `it's recommended` bit in brackets is needed. Generally, we order things in a tldr-page in the order that one would execute them. Additionally, it's rather similar to the `apt update` paradigm that's used in multiple other package managers.

What do you think?

Reply:
Well, I agree that it's unnecessary. I put it there to mimic what was done on the `apt-get.md` page. Should I keep it or remove it?